docs/examples/gradle.yml

```diff
@@ -0,0 +1,20 @@
+excludes:
+  scopes:
+  - name: "testCompile"
```

Comment:
You could have shown the power of using regular expressions for scope excludes here, which would make the file much shorter: `test.*`
Gradle defines far more scopes than all other package managers, so it's the best use case for regular expression.
Adding a separate example for an Android Gradle project would be nice, as Android projects have even more scopes. Regex would be something like `.*Test.*`.

Reply:
* I think the exact scope names are not important, because all scopes matching `test.*` are test scopes in a default project. If you don't use regex, some scopes are missing, e.g. `testApi`.
* I would consider `test.*` a Gradle specific regex.
* I would consider the risk that `test.*` matches a non-test scope very low, actually lower than accidentally missing a test scope when listing them manually.
* Yes, I would put the Android example in a separate file. But we can also add that later, not necessarily in this PR.

Reply:
Should be all addressed now.